src/Checkout/Mixin.js

```diff
@@ -413,6 +427,8 @@ export default {
             newCart.billing = Tell.serverVariable(`checkout.billing.${checkoutId}`)
             newCart.user = Tell.serverVariable(`checkout.user.${checkoutId}`)
 
+            this.activeCartCollection = newCart
```

Comment:
This seems odd when we just created `newCart` from a clone of `activeCartCollection`? Might need to discuss potential implications.